docs/docs/Project.toml

```diff
@@ -0,0 +1,2 @@
+[deps]
+PiccoloDocsTemplate = "a90a139f-c522-4b23-980b-4210ddb8d065"
```

Comment:
remove this file - curious how this got here, probably from the `./docs/get_docs_utils.sh` script but I thought we protected against that